### PR TITLE
Release 3.11.1

### PR DESCRIPTION
**Pull request by bdraco**
https://github.com/aio-libs/aiohttp/actions/runs/11840386977
Screenshot 2024-11-14 at 9 30 44 AM

### CODE REVIEW

**Comment by bdraco** [comment]
changelog message didn't render well since `RequestInfo` is non surprisingly undocumented

File: CHANGES.rst [hunk 1]
```diff
@@ -10,6 +10,24 @@
 
 .. towncrier release notes start
 
+3.11.1 (2024-11-14)
+===================
+
+Bug fixes
+---------
+
+- Added a backward compatibility layer to `~aiohttp.RequestInfo` to allow creating these objects without a `real_url` -- by :user:`bdraco`.
```
Comment (bdraco) [inline CHANGES.rst]:
```suggestion
- Added a backward compatibility layer to ``RequestInfo`` to allow creating these objects without a `real_url` -- by :user:`bdraco`.
```

File: CHANGES.rst [hunk 2]
```diff
@@ -10,6 +10,24 @@
 
 .. towncrier release notes start
 
+3.11.1 (2024-11-14)
+===================
+
+Bug fixes
+---------
+
+- Added a backward compatibility layer to ``RequestInfo`` to allow creating these objects without a `real_url` -- by :user:`bdraco`.
```
Comment (bdraco) [inline CHANGES.rst]:
```suggestion
- Added a backward compatibility layer to :class:`aiohttp.RequestInfo` to allow creating these objects without a `real_url` -- by :user:`bdraco`.
```

File: CHANGES.rst [hunk 3]
```diff
@@ -10,6 +10,24 @@
 
 .. towncrier release notes start
 
+3.11.1 (2024-11-14)
+===================
+
+Bug fixes
+---------
+
+- Added a backward compatibility layer to :class:`aiohttp.RequestInfo` to allow creating these objects without a `real_url` -- by :user:`bdraco`.
```
Comment (webknjaz) [inline CHANGES.rst]:
```suggestion
- Added a backward compatibility layer to :class:`aiohttp.RequestInfo` to allow creating these objects without a ``real_url`` -- by :user:`bdraco`.
```

Comment (bdraco) [reply CHANGES.rst]: Thanks 👍

**Pull request merged**